packages/app/src/features/liquidity-flow/components/borrow-first-position-flow/lend-step-modal/lend-step-modal.tsx

```diff
@@ -72,7 +72,8 @@ export function LendStepModal({
 
     const finalValue = Number(inputValue) > coreInputMax ? coreInputMax : inputValue
     setCoreValue(String(finalValue))
-  }, [coreDepositInfo.discount, coreDepositInfo.priceInUsd, coreInputMax, debtUsd])
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Let's leave a comment when disabling an ESLint rule so that we show thoughtfulness in our solution.

Reply:
Added